docs/rfcs/8-add-tcp-pair.md

```diff
@@ -0,0 +1,90 @@
+- Author: bokket  [bokkett@gmail.com](mailto:bokkett@gmail.com)
+- Start Date: 2021-07-17
+- RFC PR: [beyondstorage/go-endpoint#8](https://github.com/beyondstorage/go-endpoint/pull/8)
+- Tracking Issue: [beyondstorage/go-endpoint/issues/7](https://github.com/beyondstorage/go-endpoint/issues/7)
```

Comment:
I think #7 is not a tracking issue, we need to create a new one after this RFC got approved.

Reply:
ok

Reply:
```suggestion
- Tracking Issue: [beyondstorage/go-endpoint/issues/9](https://github.com/beyondstorage/go-endpoint/issues/9)
```